src/main/java/at/tugraz/sw20asd/lang/ui/Controller/StudyVocab.java

```diff
@@ -0,0 +1,226 @@
+package at.tugraz.sw20asd.lang.ui.Controller;
+
+import at.tugraz.sw20asd.lang.dto.EntryDto;
+import at.tugraz.sw20asd.lang.dto.VocabularyDetailDto;
+import at.tugraz.sw20asd.lang.ui.VocabularyAccess;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Insets;
+import javafx.scene.control.*;
+import javafx.scene.layout.*;
+import javafx.scene.layout.VBox;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import javafx.scene.paint.Color;
+
+public class StudyVocab extends VBox {
+    @FXML
+    private VBox given_list;
+    @FXML
+    private VBox answer_list;
+    @FXML
+    private VBox show_answer_list;
+    @FXML
+    private Button switch_btn;
+    @FXML
+    private Button return_btn;
+    @FXML
+    private Button show_all_btn;
+    @FXML
+    private Label study_label;
+    @FXML
+    private AnchorPane anchor_pane;
+    @FXML
+    private Label user_info;
+    @FXML
+    private Label choice_label;
+    @FXML
+    private Label given_label;
+    @FXML
+    private ComboBox<String> language_choice;
+    @FXML
+    private Button start_btn;
+
+    private List<Button> answer_button_list = new ArrayList<Button>();
+    private List<Label> given_label_list = new ArrayList<Label>();
+    private List<Label> answer_label_list = new ArrayList<Label>();
+    private List<EntryDto> words = new ArrayList<>();
+
+    private VocabularyAccess vocab;
+    private List<VocabularyDetailDto> voc;
```

Comment:
If it is inconvenient, you do not have to use any `*Dto` (= data transfer object) classes in the UI but could rather think about using dedicated model classes with properties that support attached listeners that trigger on any changes to the properties' values.